polyply/data/oplsaaLigParGen/links_PTMA.ff

```diff
@@ -119,9 +123,13 @@ resname "CH3a|PTMA"
 ; Termination of PTMA polymer with a CH3 group on the "beta" carbon
 ;--------------------------------------------------------------------------
 [link]
-resname "PTMA|CH3b"
+resname "PTMA|CH3"
 [ atoms ]
-  C17  {"replace": {"charge": 0.1111}} 
+  C17  {"replace": {"charge":  0.1111}} 
+  -C1  {"replace": {"charge": -0.1854}}
+  -H2  {"replace": {"charge":  0.0386}}
+  -H3  {"replace": {"charge":  0.0386}}
+  -H4  {"replace": {"charge":  0.0386}}
```

Comment:
@ricalessandri I think you would want to have these as +C1 and +H2 etc. Now they are referring to the previous residue, which I think is incorrect.

Reply:
yes, indeed